pkgs/servers/swego/default.nix

```diff
@@ -0,0 +1,38 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "swego";
+  version = "0.7";
+
+  src = fetchFromGitHub {
+    owner = "nodauf";
+    repo = "Swego";
+    rev = "v${version}";
+    sha256 = "14qww4ndvrxmrgkggr8hyvfpz2v7fw0vq6s8715mxa28f8pfi78b";
+  };
+
+  vendorSha256 = "068drahh0aysrm8cr5pgik27jqyk28bsx5130mc2v3xd0xmk8yp1";
+
+  buildPhase = ''
+    go build -ldflags="-s -w" -o $pname src/webserver.go
```

Comment:
Please do not overwrite buildPhase. Please use buildFlagsArray.

Reply:
Removed